test/run

```diff
@@ -78,7 +78,7 @@ testPython2() {
 
 testPython3() {
   compile "python3"
-  assertCaptured "python-3.6.2"
+  assertCaptured "python-3.6.3"
```

Comment:
I think the `testPipenvVersion` testcase in this file also needs updating (however I'd leave `testPipenvFullVersion` as-is, since that's useful to test a non-default full version number works).